### PR TITLE
DROTH-2428 Fixed moving old traffic signs

### PR DESCRIPTION
**Pull request by RobertVisan**

### CODE REVIEW

File: UI/src/view/point_asset/trafficSignForm.js [hunk 1]
```diff
@@ -626,7 +627,11 @@
 
     var singleChoiceForPanels = function (property) {
       var publicId = _.head(property);
-      var propertyValue = (!_.isEmpty(_.last(property))) ? '' : _.last(property);
+      var propertyValue;
+      if (_.last(property) === 0)
```
Comment (ricardofmsimoes) [inline UI/src/view/point_asset/trafficSignForm.js:631]: Maybe switch to

if ( _.isEmpty( _.last(property) )

File: UI/src/view/point_asset/trafficSignForm.js [hunk 2]
```diff
@@ -626,7 +627,11 @@
...
+        propertyValue = getProperties(me.pointAsset.newAsset.propertyData,  'additional_panel').defaultValue[publicId];
```
Comment (ricardofmsimoes) [inline UI/src/view/point_asset/trafficSignForm.js:632]: Can't this getProperties return  undefined ?

File: UI/src/view/point_asset/trafficSignForm.js (hunk 1 above)
Comment (RobertVisan) [reply UI/src/view/point_asset/trafficSignForm.js:631]: lodash's isEmpty does not support numbers, so for example isEmpty(1) will return true

File: UI/src/view/point_asset/trafficSignForm.js (hunk 2 above)
Comment (RobertVisan) [reply UI/src/view/point_asset/trafficSignForm.js:632]: no, because this property is defined in the assetTypeConfiguration property data for traffic signs

**Pull request merged**